inspire/base/templates/accounts/settings/profile.html

```diff
@@ -0,0 +1,64 @@
+{#
+### This file is part of INSPIRE.
+## Copyright (C) 2015 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+
+{%- import "accounts/settings/helpers.html" as helpers with context %}
+{%- from "_formhelpers.html" import render_field with context %}
+{%- extends "accounts/settings/index.html" -%}
+
+
+{% block settings_body %}
+{{helpers.panel_start('Profile', icon='fa fa-user fa-fw')}}
+{% if user.note == "2" and verification_form %}
+<form method="POST" role="form">
+{{verification_form.csrf_token}}
+<div class="alert alert-warning">
+  {{ _("You have not yet verified your email address.") }} {{ verification_form.send_verification_email(class_="btn btn-default btn-xs") }}
+</div>
+<input type="hidden" name="submit" value="verification" />
+</form>
+{% endif %}
+<div class="row"><div class="col-md-12"><p></p></div></div>
+<div class="row" id="top-row">
+  <div class="col-sm-2 col-md-2 text-center" id="user-icon"><i class="fa fa-user fa-5x"></i></div>
+    <div class="col-sm-4 col-md-4">
+    {%- if current_user.given_names or current_user.family_name %}
+      <h4 class="media-heading">{{ current_user.given_names ~ " " ~ current_user.family_name}}</h4>
+    {%- else %}
+      <h4 class="media-heading">{{ current_user.nickname}}</h4>
```

Comment:
small thing, but it is nice to be consistent in the way you add spaces. Only in some lines here I can see:

1.  ```{{ current_user.given_names ~ " " ~ current_user.family_name}}```
2.  ```{{ render_field(field) }}```
3. ```{{helpers.panel_end()}}```

Please, make them all like 2.